terraform/modules/ecs/ecs.tf

```diff
@@ -37,6 +37,56 @@ resource "aws_ecs_service" "web" {
 ECS task definitions
 ======*/
 
+/* scheduled task role */
+resource "aws_iam_role" "scheduled_task_role" {
+  name = "${var.project_name}-${var.environment}-scheduled-task-role"
+
+  assume_role_policy = <<EOF
```

Comment:
Would it be possible to remain consistent with the rest of our policy definitions and move this to a file in `./terraform/policies` and load it into here via `File` eg: 

```  
policy = "${file("./terraform/policies/ecs-autoscale-role-policy.json")}"
```